<launch>

         <node pkg="stereo"  type="rectify.py" name="rectify" />
        <node pkg="stereo" type="stereo_node" name="stereo_node" />

        <node pkg="stereo" type="stereo_tf_broadcaster" name="stereo_tf_broadcaster">
        </node>
        <node pkg="rviz" type="rviz" name="rviz" />
</launch>
